app/src/models/CityInvite.ts

```diff
@@ -1,13 +1,11 @@
 import * as Sequelize from "sequelize";
 import { DataTypes, Model, Optional } from "sequelize";
-import { User, UserAttributes, UserId } from "./User";
 import { City, CityId } from "./City";
 import { UserFileCreationAttributes } from "./UserFile";
 
 export interface CityInviteAttributes {
   id: string;
-  userId?: string;
-  locode?: string;
+  cityId?: string;
```

Comment:
Add email here? Otherwise we can't show it in a list which users have been invited.